package-lock.json

```diff
@@ -3145,14 +3145,12 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
This file gets changed when running npm install locally. Please remove this file from your commit. Thanks!